docs/getting-started.rst

```diff
@@ -22,6 +22,15 @@ To validate the installation, start a Python shell or a Jupyter notebook and typ
 
 .. code-block:: python
 
-    import gettsim
-
-    gettsim.test()
+    import pandas as pd
+    from gettsim.config import ROOT_DIR
+    from gettsim.tax_transfer import tax_transfer
+    from gettsim.policy_for_date import get_policies_for_date
+
+    tax_data = yaml.safe_load((ROOT_DIR / "data" / "param.yaml").read_text())
+    year_of_policy = 2015  # We cover 1985 to 2019
+    tax_policy = get_policies_for_date(tax_policy_data, year=year)
+    tax_policy_pensions = pd.read_excel(ROOT_DIR / "data" / "pensions.xlsx").set_index(
+        "var"
+    )
+    tax_transfer(df, tax_policy, tax_policy_pensions)
```

Comment:
What is `df` supposed to be ?

Please call by name, not by location.

Reply:
> > I uploaded a preliminary code block. This includes a minimal example. This
> > would not work though, as we would need to assign the different reform functions. See
> > https://github.com/iza-institute-of-labor-economics/gettsim/blob/976923323f2368ea52395c7b2a6b2ed040270d22/gettsim/tests/test_tax_transfer.py#L62
> > 
> > .
> > There are several options:
> > ```
> > * We leave this out.
> > 
> > * We include the same code as in the test of tax_transfer.
> > 
> > * As we now have a function, which selects the state of the policy by year, we could write the if conditions in this function and then they will also be selected by year. This also would save some code.
> > ```
> > 
> > 
> > I prefer the last option!
> 
> I do not understand much of this. What is the problem?

We adapted the code, such that reforms have to be explicitly assigned and not selected internal by year. I will just implement the third option, because then we have one function, which can be called if the user decides to select on the year paradigma.

Reply:
Sorry, my bad. Let us forget about reforms at this point. Until we have a good API for that, some time will pass. 

The use case is simply: I have some dataset and want to get the taxes and transfers for a particular year.

Reply:
Changed to dataset